src/genie/core/record/record.h

```diff
@@ -37,14 +37,14 @@ namespace record {
 class Record {
  private:
     uint8_t number_of_template_segments{};    //!< @brief
+    std::vector<Segment> reads;               //!< @brief
```

Comment:
You just reorder things here right?

Reply:
Yes it now in the order of the binary representation of the record